akommons/src/main/java/com/dewarder/akommons/content/res/TypedArray.kt

```diff
@@ -37,4 +43,16 @@ inline fun <R> TypedArray.use(block: (TypedArray) -> R): R {
             recycle()
         }
     }
-}
\ No newline at end of file
+}
+
+inline fun <R> Context.useStyledAttributes(@StyleableRes attrs: IntArray, block: (TypedArray) -> R): R
+    = obtainStyledAttributes(attrs).use(block)
+
+inline fun <R> Context.useStyledAttributes(@StyleRes resid: Int, @StyleableRes attrs: IntArray, block: (TypedArray) -> R): R
+    = obtainStyledAttributes(resid, attrs).use(block)
+
+inline fun <R> Context.useStyledAttributes(set: AttributeSet, @StyleableRes attrs: IntArray, block: (TypedArray) -> R): R
+    = obtainStyledAttributes(set, attrs).use(block)
+
+inline fun <R> Context.useStyledAttributes(set: AttributeSet, @StyleableRes attrs: IntArray, @AttrRes defStyleAttr: Int, @StyleRes defStyleRes: Int, block: (TypedArray) -> R): R
```

Comment:
Please, add zeros as default parameters to `defStyleAttr` and `defStyleRes`. It will make redundant function above (with signature `Context.useStyledAttributes(set, attrs, block)`)